### PR TITLE
52 interests component

### PR DESCRIPTION
**Pull request by KaterynaHerasymiva**

### CODE REVIEW

File: frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.sass
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.sass:1]: Check file names of components))

File: frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            <div class="counter">{{outputList.length}}</div>
+            <div>Interests</div>
+        </div>
+        <div class="arrow-icon">
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor"
+                 class="bi bi-chevron-down"
+                 viewBox="0 0 16 16">
+                <path fill-rule="evenodd"
+                      d="M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z"/>
+            </svg>
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.html:15]: Move to a separate file

File: frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+@Component({
+    selector: 'app-interests-dropdown-component',
+    templateUrl: './interests-dropdown-component.component.html',
+    styleUrls: ['./interests-dropdown-component.component.sass'],
+})
+export class InterestsDropdownComponentComponent {
+    toggle: boolean = false;
+
+    inputList: IIcon[] = TagsForInterests.tags;
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.ts:13]: @Input

File: frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    @Output() outputList: string[] = [];
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.ts:15]: Make it an event, and push list of selected interests on change

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

File: frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.sass
Comment (KaterynaHerasymiva) [reply frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.sass:1]: fixed

File: frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.html (hunk 1 above)
Comment (KaterynaHerasymiva) [reply frontend/src/app/shared/components/interests-dropdown-component/interests-dropdown-component.component.html]: fixed

File: frontend/src/app/shared/components/interests-dropdown/interests-dropdown.component.sass [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    align-items: center
+    fill: $blue-200
+    stroke: $blue-200
+    stroke-width: 3px
+
+
+
+
+
+
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/interests-dropdown/interests-dropdown.component.sass:144]: Delete extra space

File: frontend/src/app/shared/components/interests-dropdown/interests-dropdown.component.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    outputList: string[] = [];
+
+    selectInterest($event: Event) {
+        const ev = $event.target as HTMLInputElement;
+        const numb = parseInt(ev.id, 10);
+        const { checked } = ev;
+
+        if (checked) {
+            this.outputList.push(this.inputList[numb].icon_name);
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/interests-dropdown/interests-dropdown.component.ts:25]: please get rid of munations

File: frontend/src/app/shared/utils/tagsForInterests.ts [hunk 6]
```diff
@@ -0,0 +1,90 @@
+import { IIcon } from '@shared/models/IIcon';
+
+export class TagsForInterests {
+    static tags: IIcon[] =
```
Comment (tatianahutii) [inline frontend/src/app/shared/utils/tagsForInterests.ts:4]:
```suggestion
export const getTags = () =>
```

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**